recipes/hsm/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+import os
+
+required_conan_version = ">=1.32.1"
```

Comment:
```suggestion
required_conan_version = ">=1.33.0"
```

strip_root requires 1.33